src/Libraries/RevitNodes/Elements/RevisionCloud.cs

```diff
@@ -0,0 +1,209 @@
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        public override Autodesk.Revit.DB.Element InternalElement
+        {
+            get { return InternalRevitElement; }
+        }
+
+        #endregion
+
+        #region Private constructors
+
+        /// <summary>
+        /// Create revision cloud from revit element
+        /// </summary>
+        /// <param name="RevisionCloud"></param>
+        private RevisionCloud(Autodesk.Revit.DB.RevisionCloud RevisionCloud)
+        {
+            SafeInit(() => InitElement(RevisionCloud));
+        }
+
+        /// <summary>
+        /// Create Revision cloud from view
+        /// </summary>
+        /// <param name="view"></param>
+        /// <param name="curves"></param>
+        /// <param name="id"></param>
+        private RevisionCloud(Autodesk.Revit.DB.View view, List<Autodesk.Revit.DB.Curve> curves, Autodesk.Revit.DB.ElementId id)
+        {
+            SafeInit(() => Init(view, curves, id));
+        }
+
+        #endregion
+
+        #region Private mutators
+
+        /// <summary>
+        /// Set the internal Element, ElementId, and UniqueId
+        /// </summary>
+        /// <param name="wall"></param>
+        private void InternalSetElement(Autodesk.Revit.DB.RevisionCloud element)
+        {
+            InternalRevitElement = element;
+            InternalElementId = element.Id;
+            InternalUniqueId = element.UniqueId;
+        }
+
+        #endregion
+
+        #region Helpers for private constructors
+
+        private void InitElement(Autodesk.Revit.DB.RevisionCloud element)
+        {
+            InternalSetElement(element);
+        }
+
+        /// <summary>
+        /// Create new RevisionCloud from Curves
+        /// </summary>
+        /// <param name="view"></param>
+        /// <param name="curves"></param>
+        /// <param name="id"></param>
+        private void Init(Autodesk.Revit.DB.View view, IEnumerable<Autodesk.Revit.DB.Curve> curves, Autodesk.Revit.DB.ElementId id)
+        {
+            // get document and open transaction
+            Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(document);
+
+            // get existing element if possible
+            var element = ElementBinder.GetElementFromTrace<Autodesk.Revit.DB.RevisionCloud>(document);
+
+
+            if (element == null)
+            {
+                element = Autodesk.Revit.DB.RevisionCloud.Create(document, view, id, curves.ToList());
+            }
+            else
+            {
+                // there is no way of updateing the properties.
+                // So the cloud is being recreated.
+                document.Delete(element.Id);
+                element = Autodesk.Revit.DB.RevisionCloud.Create(document, view, id, curves.ToList());
```

Comment:
ImportInstance just deletes the element first. Changed this in both (RevCloud & FilledRegion).